rasa/core/test.py

```diff
@@ -910,12 +1004,14 @@ async def test(
             # Add conversation level accuracy to story report.
             num_failed = len(story_evaluation.failed_stories)
             num_correct = len(story_evaluation.successful_stories)
+            num_warnings = len(story_evaluation.stories_with_warnings)
             num_convs = num_failed + num_correct
             if num_convs:
                 conv_accuracy = num_correct / num_convs
                 report["conversation_accuracy"] = {
                     "accuracy": conv_accuracy,
                     "correct": num_correct,
+                    "with_warnings": num_warnings,
```

Comment:
do we need to update the API spec for this?

Reply:
@alwx

Reply:
fixed it